CMakeLists.txt

```diff
@@ -164,7 +164,7 @@ if(KernelArchARM)
     endif()
 endif()
 if(KernelArchRiscV)
-    KernelCommonFlags(-mcmodel=medany)
+    KernelCommonFlags(-mcmodel=medany -msmall-data-limit=64 -fno-common)
```

Comment:
-fno-common should be the default setting for gcc?

Reply:
Unclear if that's the upsteam default, but if I don't have that I get a large number of `*COM*` symbols in `kernel/CMakeFiles/kernel.elf.dir/kernel_all.c.obj` and they ultimately get allocated in `.bss`, when using the `sel4-riscv` image

Reply:
Thanks for the info.

Reply:
Are you able to add a comment explaining how without `-fno-common` many symbols that would otherwise end up in the small sections (.sbss, .srodata and .sdata) get left in the default sections (.bss, .rodata and .data) so that it's clear that both options are dependent on each other?